### ISSUE TITLE
SyntaxError: Unexpected end of JSON input

### ISSUE THREAD

**Issue by sammygrey** [Jinzulen/TenorJS#10]
**Describe the bug**
"SyntaxError: Unexpected end of JSON input" returned after queries

**To Reproduce**
Using the tenorjs client while filter is set on high and locale is set to en_US some API queries will return an "Unexpected end" of the JSON Input. 
This seems to occur whenever I am calling queries in quick succession but the query term that has always replicated this issue is "lol", which works fine on the tenor website but creates this error within using the API. 

**Screenshots**
![image](https://user-images.githubusercontent.com/49354894/113069687-e9461580-918e-11eb-9291-10ee74cf1939.png)

**Additional context**
I'm using handlebars, nodemon, and express for this as part of a class project.
1.0.7 is the version I am using.

**Comment by Jinzulen** [comment]
Your instructions are really vague and as a result, I wasn't able to reproduce the error. However, as I suspected, this is the same issue that was reported by Florian Proksch [here](https://github.com/Jinzulen/TenorJS/issues/8). He's already issued a pull request with a fix that I have verified does indeed solve the problem.

I will merge and push it momentarily, it should fend off this issue for now, giving me the time and the breathing room to rewrite this messy library.
